core/src/main/java/org/mini2dx/connect4/Connect4Game.java

```diff
@@ -1,24 +1,43 @@
 package org.mini2dx.connect4;
 
-import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.graphics.GL20;
+
 import com.badlogic.gdx.graphics.Texture;
 import org.mini2Dx.core.game.BasicGame;
 import org.mini2Dx.core.graphics.Graphics;
 
+
+
 public class Connect4Game extends BasicGame {
 	public static final String GAME_IDENTIFIER = "org.mini2dx.connect4";
 
 	private Texture texture;
+	private BlueTile blueTile;
+	private RedTile redTile;
+	private Board board;
+	private EmptyTile empty;
 	
 	@Override
     public void initialise() {
-    	texture = new Texture("mini2Dx.png");
+    	texture = new Texture("panel_beigeLight.png");
```

Comment:
"panel_beigeLight.png" is a magic string and should be moved to a static final variable.